docs/code_API_mover.md

```diff
@@ -0,0 +1,50 @@
+# Mover APIs
+
+LabExT's Mover offers several possibilities to extend functionalities by user-defined code.
+This guide gives practical hints to extend the Mover.
+
+## Stages
+
+To support another Stage, a new Stage implementation must be created.
+Currently, the [following stages]() are supported in LabExT-Core.
+New stages can be included by an addon path or permanently supported in Core by creating a pull request.
+
+Each new stage must inherit from the abstract interface Stage in order to be included by LabExT:
+
+```py
+from LabExT.Movement.Stage import Stage
+
+class MyNewStage(Stage):
+    pass
+```
+
+The [Stage]() interface defines all the methods and properties that each new implementation must define to work correctly.
+Below we provide notes on the methods and properties to implement:
+
+::: LabExT.Movement.Stage.Stage
+    rendering:
+        show_root_heading: true
+        sort_members: source
```

Comment:
Somehow the names of the arguments are not printed? 

I would expect the argument name instead of just the type.

<img width="777" alt="image" src="https://user-images.githubusercontent.com/3842684/221779305-8b3b173a-4a18-422d-b4a1-b6f3118367e9.png">

Reply:
Changing the doc string style does the trick!
<img width="778" alt="Screenshot 2023-03-02 at 10 52 08" src="https://user-images.githubusercontent.com/14354617/222393914-3f2a8f95-b7f9-4178-8c5a-a6ce501267d8.png">